web/tests/Web.E2ETests/Pages/School/BenchmarkWorkforcePage.cs

```diff
@@ -36,13 +36,24 @@ public class BenchmarkWorkforcePage(IPage page)
     private ILocator SaveAsImageButtons =>
         page.Locator(Selectors.Button, new PageLocatorOptions { HasText = "Save as image" });
 
+    private ILocator HowWeChooseSimilarSchoolDetailsBtn =>
+        page.Locator(Selectors.GovDetailsSummaryText, new PageLocatorOptions { HasText = "How we choose similar schools" });
+
+    private ILocator ViewOrChangeSchoolComparatorLink => page.Locator(Selectors.GovLink,
+        new PageLocatorOptions { HasText = "View or change which schools we compare you with" });
+
+    private ILocator HowWeChooseSimilarSchoolsDetailsSection => page.Locator(Selectors.GovDetailsText);
+
     public async Task IsDisplayed()
     {
         await PageH1Heading.ShouldBeVisible();
         await Breadcrumbs.ShouldBeVisible();
         await ChangeSchoolLink.ShouldBeVisible();
         await ViewAsTableRadio.ShouldBeVisible().ShouldBeChecked(false);
         await ViewAsChartRadio.ShouldBeVisible().ShouldBeChecked();
+        await HowWeChooseSimilarSchoolDetailsBtn.ShouldBeVisible();
```

Comment:
This isn't a button, and therefore shouldn't be named as such

Reply:
updated it now.